pywinauto/win32defines.py

```diff
@@ -34,23 +34,21 @@
 
 """Definition of Windows defines"""
 from __future__ import unicode_literals
+from ctypes import c_int
 
 __revision__ = "$Revision$"

```

Comment:
I think these `__revision__` lines could be removed. It's inherited from Mercurial or SVN probably.